src/main/groovy/de/dkfz/roddy/execution/jobs/GenericJobInfo.groovy

```diff
@@ -6,20 +6,47 @@
 
 package de.dkfz.roddy.execution.jobs
 
-
+import de.dkfz.roddy.config.ResourceSet
 import de.dkfz.roddy.tools.BufferUnit
 import de.dkfz.roddy.tools.TimeUnit
 import groovy.transform.CompileStatic
 
+import java.time.Duration
+import java.time.LocalDateTime
+
 /**
  * Created by michael on 06.02.15.
  */
 @CompileStatic
 class GenericJobInfo {
 
```

Comment:
I would be great if we could also get the values for

- eligible time (etime in PBS)
- account name (Account_Name in PBS)
- start count (start_count in PBS)